code/modules/projectiles/gun_system.dm

```diff
@@ -800,7 +799,9 @@ and you're good to go.
 	//^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 	shots_fired++
-
+	// Dumb akimbo code
+	if(dual_wield && (!gun_user.get_active_held_item().current_mag || gun_user.get_active_held_item().current_mag.current_rounds <= 0))
+		gun_user.shoot_inactive_hand = TRUE
```

Comment:
You want to do the same for when gun_user.shoot_inactive_hand is FALSE, to make a cycle

Reply:
Yea but I am still working on it and can't make it work, cause i am bad